packages/core/test/utils_test.js

```diff
@@ -1770,6 +1770,21 @@ describe("utils", () => {
       });
     });
 
+    it("should handle null formData for schema which contains additionalProperties", () => {
```

Comment:
@epicfaace do you still want to keep this test?

Reply:
Sure